ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,206 @@
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "relationshipToChild",
+    "FieldType": "TextArea",
+    "ElementLabel": "What is the respondent’s relationship to the child or children in this case?",
+    "HintText": "Include: the name of the child or children, the respondent’s relationship to them and any details if you’re not sure this person has parental responsibility",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "respondent_border_2",
+    "FieldType": "Label",
+    "ElementLabel": "-------------------------------------------------------------------------------------------------------------",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "contactDetailsHidden",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Do you need contact details hidden from anyone?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "contactDetailsHiddenReason",
+    "FieldType": "TextArea",
+    "ElementLabel": "Give reason",
+    "FieldShowCondition": "contactDetailsHidden=\"Yes\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "proceedingsLabel",
+    "FieldType": "Label",
+    "ElementLabel": "## Ability to take part in proceedings",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "litigationIssues",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "LitigationCapacityIssues",
+    "ElementLabel": "Do you believe this person will have problems with litigation capacity (understanding what's happening in the case)?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "litigationIssuesDetails",
+    "FieldType": "TextArea",
+    "ElementLabel": "Give details, including assessment outcomes and referrals to health services",
+    "FieldShowCondition": "litigationIssues=\"YES\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "party",
+    "ListElementCode": "respondent_border_3",
+    "FieldType": "Label",
+    "ElementLabel": "-------------------------------------------------------------------------------------------------------------",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyRespondent",
+    "ListElementCode": "leadRespondentIndicator",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Lead Respondent Indicator",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyRespondent",
+    "ListElementCode": "party",
```

Comment:
I think we should rename that complex type to `RespondentParty` because obviously we will keep it when migration is done and generic name won't work well when we have other instances of party for applicant or others.

Reply:
makes sense to me 👍